### PR TITLE
Update 2020-09-21-laves20a.md

### PR DESCRIPTION
**Pull request by mlaves**
Added correct umlaut to one of the authors.

### CODE REVIEW

File: _posts/2020-09-21-laves20a.md [hunk 1]
```diff
@@ -25,15 +25,15 @@ page: 393-412
 order: 393
 cycles: false
 bibtex_author: Laves, Max-Heinrich and Ihler, Sontje and Fast, Jacob F. and Kahrs,
-  Luder A. and Ortmaier, Tobias
+  Lüder A. and Ortmaier, Tobias
```
Comment (lawrennd) [inline _posts/2020-09-21-laves20a.md]: `L\"uder A. and Ortmaier, Tobias`

You need to use latex for the umlaut here.

**Review by lawrennd — CHANGES_REQUESTED** [review]
Thanks can you use latex for the umlaut in the bibtex_author field? Thanks!

**Comment by mlaves** [comment]
I changed the entry as per suggestion.

**Review by lawrennd — APPROVED** [review]
Great, thanks Max-Heinrich!

**Pull request merged**